Dockerfile.redhat

```diff
@@ -117,7 +117,7 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
             http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/libusbx-devel-1.0.23-4.el8.x86_64.rpm && \
             yum clean all
 
-RUN python3 --version && python3 -m pip install numpy==1.21.0 --no-cache-dir
+RUN python3 --version && python3 -m pip install numpy==1.26.4 --no-cache-dir
```

Comment:
it would be safer to use a rule <2.0.0 in case older like 3.8 doesn't include it. It might fail on ubuntu20.

Reply:
It does not work:
+ python3 -m pip install numpy --no-cache-dir
/bin/bash: line 1: =2.0.0: No such file or directory

Reply:
"numpy<2.0.0" add quotes